client/src/App.vue

```diff
@@ -1,15 +1,38 @@
 <template>
```

Comment:
do not modify the existing component. Create a new component/route and add your changes there.